modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/info/item/provider/CalendarBookingInfoItemFieldValuesProvider.java

```diff
@@ -67,13 +70,18 @@ calendarBooking, _getThemeDisplay())
 					calendarBooking.getCalendarBookingId())
 			).build();
 		}
+		catch (PortalException portalException) {
+			throw new RuntimeException(
+				"Caught unexpected exception", portalException);
+		}
```

Comment:
Why do you need a catch for PortalException? It handles the error in the same way as the generic Exception.

Reply:
I was copying the pattern in other *FieldValuesProvider:

https://github.com/liferay/liferay-portal/blob/master/modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/info/item/provider/FileEntryInfoItemFieldValuesProvider.java#L89-L95

https://github.com/liferay/liferay-portal/blob/master/modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/info/item/provider/JournalArticleInfoItemFieldValuesProvider.java#L117-L123

If it's not useful though I can remove it.

Reply:
Sounds good